webapp/src/js/services/db.js

```diff
@@ -87,5 +87,4 @@ angular.module('inboxServices').factory('DB',
     }
 
     return get;
-  }
-);
```

Comment:
Consider reverting this file - nothing actually changed, right?

Reply:
Yeah! I wanted to at the time but my energy drained when I realised to do it I either had to:
 - Rebase from the start and try to edit out all the changes
 - Revert it, disable prettier and *then* commit the revert, because otherwise prettier will pretty up my de-prettying.

I'll work it out and fix it.